test/api_integration/apis/saved_objects/bulk_get.js

```diff
@@ -94,6 +95,7 @@ export default function ({ getService }) {
                     buildNum: 8467,
                     defaultIndex: '91200a00-9efd-11e7-acb3-3dab96693fab',
                   },
+                  namespaces: ['default'],
```

Comment:
This test ensures that we return a `namespaces` field for a namespace-agnostic object -- do we want to do that?

Reply:
My bad! I blame that comment on lack of coffee 😅